outport/disabledOutport.go

```diff
@@ -0,0 +1,57 @@
+package outport
+
+import (
+	"github.com/ElrondNetwork/elrond-go-core/data"
+	"github.com/ElrondNetwork/elrond-go-core/data/indexer"
+)
+
+type disabledOutport struct{}
+
+// NewDisabledOutport will create a new instance of disabledOutport
+func NewDisabledOutport() OutportHandler {
```

Comment:
should return `*disabledOutport` instead of the interface in order to comply with the project